docs/emotion-utils.md

```diff
@@ -10,6 +10,7 @@ A collection of Emotion utility functions.
 
 |  Variable | Description |
 |  --- | --- |
+|  [createBreakpointStyles](./emotion-utils.createbreakpointstyles.md) | Create function that applies styles when viewport width is greater than or equal to and/or less than the specified breakpoints. |

```

Comment:
I decided to remove the Fresnel code since a) it removes the need for us to worry about JSX in this package; and b) it’s only a few lines of code to call Fresnel’s `createMedia` in the projects. Because of this I rename `createMedia` to `createBreakpointStyles`.